custom_metric.py

```
@@ -0,0 +1,329 @@
+from deepeval.test_case import LLMTestCase
+from typing import Optional, List, Union
+
+from deepeval.utils import get_or_create_event_loop, prettify_list
+from deepeval.metrics.utils import (
+    construct_verbose_logs,
+    trimAndLoadJson,
+    check_llm_test_case_params,
+    initialize_model,
+)
+from deepeval.test_case import (
+    LLMTestCase,
```

Ruff (0.8.2)

Redefinition of unused `LLMTestCase` from line 3

Remove definition: `LLMTestCase`

(F811)